config/taxonomy/policymanager_response.structs.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "opa.schema.json",
+    "title": "OPA Data Response Taxonomy",
+    "description": "This taxonomy defines the structure of response received from Policy Agent policy engine with enforcements.",
+    "type": "object",
+    "properties": {
+        "decision_id": {
+            "description": "This is the id returned by the governance engine",
+            "type": "string"
+        },
+        "result": {
+            "type": "array",
+            "description": "While showing the result, action contains the action type and the associated entity on which action has been taken.",
+            "items": {
+                "properties": {
+                    "action": { "$ref": "module.structs.schema.json#/definitions/action" },
+                    "used_policy": {
+                        "description": "The list of policies on which the decision was based.",
+                        "type": "string"
+                    }
```

Comment:
This is different than the design doc because it would require rules to look like:
```rego
rule[{"used_policy": "Redact PII columns for CBA", "action": {"name": "RedactColumn", "columns": columns}] {
  ...
}
```

Instead of:
```rego
rule[{"description": "Redact PII columns for CBA", "action": "RedactColumn", "columns": columns}] {
  ...
}
```

The differences are:
1. `used_policy` instead of `description` but we already discussed it and said that the change is fine. I would consider changing it to "policy" instead of "used_policy" though.
2. `action` is a structure instead of being flat

I'm not asking for a change but rather for you to consider if this is what is desired in terms of how people would write opa policies. Thanks.

Reply:
Changed used_policy to policy. For this I prefer to keep it in the format:
rule[{"used_policy": "Redact PII columns for CBA", "action": {"name": "RedactColumn", "columns": columns}] {
  ...
}
Because, an action here us corresponds to the name of that action and the columns on which the action has been made. Seems to be a clearer view of the action. @roee88